store/templates/registration/login.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"></script>
+</head>
+<body>
+    <div class="form-group flex-column justify-content-around align-content-center d-flex" style="width: 100vw; height:100vh">
+        <div style="width: 50%; align-self:center">
+            <div class="alert alert-danger alert-dismissible collapse" id='error'>
+                <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+                <strong>Ah!</strong> Username of password did not match</div>
+        <form method="POST">
+            {% csrf_token %}
+            <input type="text" class="form-control p-2" name="Username" placeholder="Username" id="Username" required><br>
+            <input type="password" class="form-control p-2" name="password" placeholder="Password" id="password" required><br>
```

Comment:
Better keep the name and id field to be consistent everywhere (all lowercase would be fine). However, this is fine for this assignment.

Reply:
Yes i messed it up in one place then dodnt changed it and realised that the id was comming alot of places .